templates/compute-domain-daemon-claim-template.tmpl.yaml

```diff
@@ -8,6 +8,7 @@ metadata:
     - {{ .Finalizer }}
   labels:
     {{ .ComputeDomainLabelKey }}: {{ .ComputeDomainLabelValue }}
+    app: {{ .AppLabelValue }}
```

Comment:
aestheitcally, this probably makes sense as the first label in all of these templates